src/rtcModule/webrtc.cpp

```diff
@@ -585,6 +607,53 @@ void RtcModule::hangupAll(TermCode code)
     }
 }
 
+void RtcModule::launchCallRetry(Id chatid, AvFlags av, bool starter)
```

Comment:
It would be a good idea for readability to refactor `starter` --> `isActiveRetry`